exporter/util.go

```diff
@@ -240,6 +247,7 @@ func (ic *importContext) emitNotebookOrRepo(path string) {
 	if strings.HasPrefix(path, "/Repos") {
 		ic.emitRepoByPath(path)
 	} else {
+		// TODO: strip /Workspace prefix if it's provided
```

Comment:
Question: Is this TODO not required currently? what happens if /Workspace prefix is provided?

Reply:
it's related to this issue: https://github.com/databricks/terraform-provider-databricks/issues/3195 - we're introduced ability to refer to objects with `/Workspace`  and without, but notebooks/workspace files right now are using path without `/Workspace`, so I just left a comment for myself to handle this case in the near future